.github/workflows/auto-cherry-picker.yaml

```diff
@@ -84,14 +83,13 @@ jobs:
     name: Post-Pick Actions
     needs: [cherry_picker, prerequisites]
     runs-on: ubuntu-latest
-    if: always()
+    if: success() || failure()
```

Comment:
Is the theory that if `prerequisites` is skipped, the workflow status won't be `success` or `failure` and so this won't run?

What happens if the workflow _isn't_ skipped (as in, a PR does need cherry picking)? Given the error is something about tokens, the root cause sounds like it might apply to that situation too (e.g. we're providing an incorrect token or something). Am I misunderstanding something?

Reply:
You got it. I don't think skip is success or failure, so although I couldn't be 100% sure this'll work (and unfortunately an act bug stopped me from testing it), I have mediumly high confidence.

As to tokens, yeah sorry the real fix is switching github-script to upstream.